### PR TITLE
Update maxhitplugin

### PR DESCRIPTION
**Pull request by Seebity**
Refactored version with fixes

### CODE REVIEW

**Comment by iProdigy** [comment]
would recommend including https://github.com/Matt12yearslater/max-hit/pull/31 as well

File: plugins/maxhitplugin [hunk 1]
```diff
@@ -1,2 +1,2 @@
 repository=https://github.com/Matt12yearslater/max-hit.git
-commit=23d328cb160dd471431fd88fbb8268b8f01f01a2
+commit=5edb716712f06a8dd3de933d414f7629ff19a24c
```
Comment (iProdigy) [inline plugins/maxhitplugin:2]:
```suggestion
commit=5edb716712f06a8dd3de933d414f7629ff19a24c
authors=Matt12yearslater,Seebity
```

@Matt12yearslater confirming you are okay with Seebity being able to update the commit hash?

**Comment by Seebity** [comment]
Just an FYI I made the changes in the repo with the wrong account (@caleblee9) but Matt invited me to collaborate on this account (see https://github.com/runelite/plugin-hub/pull/6512)

**Comment by Alexsuperfly** [comment]
Can you open this with that account then or have that account add this one as an author, because i have no way to verify whos who and what rights people have to the repo other than the commit history to the repo itself

**Comment by Seebity** [comment]
@Alexsuperfly @iProdigy Anything else I need to do before review?

**Comment by caleblee9** [comment]
Opened https://github.com/runelite/plugin-hub/pull/6542 with other account

**Pull request closed without merging**